src/sentry/integrations/pagerduty/integration.py

```diff
@@ -0,0 +1,185 @@
+from __future__ import absolute_import
+
+from django.utils.translation import ugettext_lazy as _
+from django.db import transaction
+
+from sentry import options
+
+from sentry.utils import json
+from sentry.integrations import (
+    IntegrationInstallation,
+    IntegrationFeatures,
+    IntegrationMetadata,
+    IntegrationProvider,
+    FeatureDescription,
+)
+from sentry.models import OrganizationIntegration, PagerDutyServiceProject, PagerDutyService, Project
+from sentry.pipeline import PipelineView
+
+DESCRIPTION = """
+PagerDuty Description
+"""
+
+FEATURES = [
+    FeatureDescription(
+        """
+        Configure rule based PagerDuty notifications!!
```

Comment:
Is this copy changing?

Reply:
This will definitely need to be changed - but I have it down as part of the checklist of things to update before GA release